### PR TITLE
all: fix staticcheck warnings

### PR DESCRIPTION
**Pull request by johanbrandhorst**

### CODE REVIEW

File: internal/admin/authorizedapps/form.go [hunk 1]
```diff
@@ -73,11 +73,11 @@ func (f *formData) PopulateAuthorizedApp(a *model.AuthorizedApp) error {
 	var err error
 	a.SafetyNetPastTime, err = time.ParseDuration(f.SafetyNetPastTime)
 	if err != nil {
-		return fmt.Errorf("Failed to parse durection for SafetyNetPastTime: %w", err)
+		return fmt.Errorf("failed to parse durection for SafetyNetPastTime: %w", err)
```
Comment (mikehelmick) [inline internal/admin/authorizedapps/form.go:76]: s/durection/duration/

File: internal/admin/authorizedapps/form.go [hunk 2]
```diff
@@ -73,11 +73,11 @@ func (f *formData) PopulateAuthorizedApp(a *model.AuthorizedApp) error {
...
 	}
 	a.SafetyNetFutureTime, err = time.ParseDuration(f.SafetyNetFutureTime)
 	if err != nil {
-		return fmt.Errorf("Failed to parse durection for SafetyNetFutureTime: %w", err)
+		return fmt.Errorf("failed to parse durection for SafetyNetFutureTime: %w", err)
```
Comment (mikehelmick) [inline internal/admin/authorizedapps/form.go:80]: s/durection/duration/

sorry - these are my spelling errors

File: internal/admin/authorizedapps/form.go (hunk 1 above)
Comment (sethvargo) [reply internal/admin/authorizedapps/form.go:76]: durection is just a "signed" duration (a duration with direction 😄)

**Pull request merged**